python/dlr/tf_model.py

```diff
@@ -0,0 +1,154 @@
+# coding: utf-8
+import logging
+import os
+import tensorflow as tf
+
+# A prefix that will be prepended to the names in graph_def
+PREFIX = "import"
+
+
+def _load_frozen_graph(frozen_graph_file, device):
```

Comment:
_load_frozen_graph -> load_frozen_graph, would be good to add description

Reply:
added docs